diagnostic.md

```diff
@@ -5,7 +5,7 @@ Place your responses inside the fenced code-blocks where indivated by comments.
 1.  Describe a reason why a join tables may be valuable.
 
   ```md
-    # < Your Response Here >
+    so that you know which tables are related
```

Comment:
Make sure to note that join tables are specifically for many-to-many relationships